<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPane</class>
 <widget class="QWidget" name="ColorPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonSelectFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Slect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonClearFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="labelFieldFileName">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelDataInfo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Data info:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFieldDataInfo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUseFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Use this file</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxLogScale">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Log scale</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxInterpolation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>interpolate</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxUserDefinedRange">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>User defined range</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Range:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditRangeMin"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditRangeMax"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLegend">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Legend</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Title: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditLegendTitle"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxFontSize"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Number of labels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxNumTics">
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxBarWidth">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxBarHeight">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radioButtonScalarBarV">
          <property name="text">
           <string>Vertical</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonScalarBarH">
          <property name="text">
           <string>Horizontal</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>314</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSelectFile</sender>
   <signal>pressed()</signal>
   <receiver>ColorPane</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClearFile</sender>
   <signal>pressed()</signal>
   <receiver>ColorPane</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseFile</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ColorPane</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClearFile</sender>
   <signal>pressed()</signal>
   <receiver>ColorPane</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRangeMin</sender>
   <signal>editingFinished()</signal>
   <receiver>ColorPane</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRangeMax</sender>
   <signal>editingFinished()</signal>
   <receiver>ColorPane</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLogScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>ColorPane</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxInterpolation</sender>
   <signal>toggled(bool)</signal>
   <receiver>ColorPane</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFile()</slot>
  <slot>clear()</slot>
  <slot>setChanged()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
